packages/api/src/routes/auth/events/createLinkAccountEvent.js

```diff
@@ -25,11 +25,12 @@ function createLinkAccountEvent(context, { authConfig, plugins }) {
 
   if (linkAccountPlugins.length === 0) return undefined;
 
-  async function linkAccountEvent({ account, user }) {
+  async function linkAccountEvent({ account, provider, user }) {
     for (const plugin of linkAccountPlugins) {
       await plugin.fn({
         properties: plugin.properties ?? {},
         account,
+        provider,
```

Comment:
Sorry, this should be `profile` and not `provider`

Reply:
Fixed. Sorry.